packages/scaffold-ui/src/views/w3m-account-settings-view/styles.ts

```diff
@@ -1,50 +1,3 @@
 import { css } from 'lit'
 
-export default css`
-  wui-flex {
-    width: 100%;
-  }
-
-  wui-icon-link {
-    margin-right: calc(var(--wui-icon-box-size-md) * -1);
-  }
-
-  .account-links {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-  }
-
-  .account-links wui-flex {
-    cursor: pointer;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex: 1;
-    background: red;
-    align-items: center;
-    justify-content: center;
-    height: 48px;
-    padding: 10px;
-    flex: 1 0 0;
-
-    border-radius: var(--XS, 16px);
-    border: 1px solid var(--dark-accent-glass-010, rgba(71, 161, 255, 0.1));
-    background: var(--dark-accent-glass-010, rgba(71, 161, 255, 0.1));
-    transition: background-color var(--wui-ease-out-power-1) var(--wui-duration-md);
-    will-change: background-color;
-  }
-
-  .account-links wui-flex:hover {
-    background: var(--dark-accent-glass-015, rgba(71, 161, 255, 0.15));
-  }
-
-  .account-links wui-flex wui-icon {
-    width: var(--S, 20px);
-    height: var(--S, 20px);
-  }
-
-  .account-links wui-flex wui-icon svg path {
-    stroke: #47a1ff;
-  }
-`
+export default css``
```

Comment:
If this file is no longer used I'd remove it directly. 
btw where did all of these styles go?